examples/globe.html

```diff
@@ -81,17 +78,25 @@
                 miniView.notifyChange(camera);
             });
 
-            // Add one imagery layer to the miniview
-            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(layer) { miniView.addLayer(layer); });
 
-            // Add one imagery layer to the scene
+            // Add one imagery layer to the scene and the miniView
             // This layer is defined in a json file but it could be defined as a plain js
             // object. See Layer* for more info.
-            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(addLayerCb);
+            itowns.Fetcher.json('./layers/JSONLayers/Ortho.json').then(function _(config) {
+                var layer = new itowns.ColorLayer(config.id, config);
+                view.addLayer(layer).then(menuGlobe.addLayerGUI.bind(menuGlobe));
+                miniView.addLayer(layer);
```

Comment:
Maybe do 2 layers, because visibility or opacity layer

Reply:
Ok, but we could have an other example that shows on layer in multiple view

Reply:
yes